samples/outlook/90-other-item-apis/get-diagnostic-information.yaml

```
@@ -10,17 +10,20 @@ script:
         $("#get-diagnostics").click(getDiagnostics);
 
         function getDiagnostics() {
-          // This snippet gets a mailbox's diagnostic information, such as host application and version, and prints it to the console. 
+          // This snippet gets a mailbox's diagnostic information, such as Outlook client and version, and logs it to the console. 
           const diagnostics = Office.context.mailbox.diagnostics;
-          console.log(`Host application: ${diagnostics.hostName}`);
-          console.log(`Host version: ${diagnostics.hostVersion}`);
+          console.log(`Outlook client: ${diagnostics.hostName}`);
```

I think hostName will return Outlook*
```suggestion
          console.log(`Client application: ${diagnostics.hostName}`);
```